lib/job.js

```diff
@@ -231,17 +231,19 @@ class Job extends BaseModel {
 
         try {
             const newPeriodic = getAnnotations(newJob.permutations[0], 'screwdriver.cd/buildPeriodically');
-            const isJobEnabled = newJob.state === 'ENABLED';
-            const isJobArchived = newJob.archived;
+            const newIsJobEnabled = newJob.state === 'ENABLED';
+            const oldIsJobEnabled = oldJob.state === 'ENABLED';
```

Comment:
It looks there are two kind of naming rule (`oldIsJobEnabled` and `wasJobEnabled`).
I think these names should be unified because both are based on `oldJob` properties.

Reply:
Fixed to use `isOldJobEnabled` instead of `wasJobEnabled`